packs/equipment-effects/effect-fury-cocktail-moderate.json

```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Fury Cocktail (Moderate)",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Fury Cocktail (Moderate)]</p>\n<p><strong>Benefit</strong> You gain a +3 item bonus to melee attack rolls and an additional effect depending on the additive chosen when the brew is created.</p><ul><li><strong>Animalistic</strong> choice between a jaws attack that deals 1d6 piercing damage or a claw attack that has the agile trait and deals 1d4 slashing damage.</li><li><strong>Double</strong> resistance 2 to physical damage</li><li><strong>Mournful</strong> resistance 5 to void or vitality depending on void healing</li><li><strong>Skeptical</strong> +3 item bonus to saves against magic</li><li><strong>Titanic</strong> size change to Large, Clumsy 1, and change of reach (+5 for Medium and Small, +10 for Tiny)</li><li><strong>Wyrmhide</strong> resistance 5 to acid, cold, electricity, fire, and poison.</li></ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Fury Cocktail (Moderate)]</p>\n<p><strong>Benefit</strong> You gain a +3 item bonus to melee attack rolls and an additional effect depending on the additive chosen when the brew is created.</p><ul><li><strong>Animalistic</strong> choice between a jaws attack that deals 1d6 piercing damage or a claw attack that has the agile trait and deals 1d4 slashing damage.</li><li><strong>Double</strong> resistance 2 to physical damage</li><li><strong>Mournful</strong> resistance 5 to void or vitality depending on void healing</li><li><strong>Skeptical</strong> +3 item bonus to saves against magic</li><li><strong>Titanic</strong> size change to Large, Clumsy 1, and change of reach (+5 for Medium and Small, +10 for Tiny)</li><li><strong>Wyrmhide</strong> resistance 5 to acid, cold, electricity, fire, and poison.</li></ul><p><strong>Drawback</strong> You take a -1 penalty to AC and a -2 penalty to Reflex saves.</p>"
```

Comment:
Oh, that's the occasion for me to ask:
why do you guys prefer the 'en dash' unicode character (8211) instead of the 'hyphen-minus' character (45) ? There must be a reason.